packages/pn-commons/src/utility/paymentCaching.utility.ts

```diff
@@ -0,0 +1,126 @@
+import _ from 'lodash';
+
+import { PaymentCache, PaymentDetails, PaymentStatus } from '../models';
+import { paymentCacheSchema } from '../models/PaymentCache';
+
+export const PAYMENT_CACHE_KEY = 'payments';
+
+export const getPaymentCache = (): PaymentCache | null => {
+  const paymentCache = sessionStorage.getItem(PAYMENT_CACHE_KEY);
```

Comment:
Did you try to put malicious code into sessionStorage and check what happen?

Reply:
Yes, it throw an error/crash